tools/i18n/build.py

```
@@ -0,0 +1,373 @@
+    if (local / f"{locale}.json").exists():
+        return local
+
+    return None
+
+
+def load_translations(locale, translations_dir=None):
+    """Load translation JSON for the given locale, keyed by file."""
+    tdir = find_translations_dir(locale, translations_dir)
+    if tdir is None:
+        print(f"Error: No translations found for locale '{locale}'.", file=sys.stderr)
+        print(f"  Searched:", file=sys.stderr)
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)